README.md

```diff
@@ -318,6 +318,34 @@ git ls-files -- '*.go' | xargs errtrace -w
 errtrace can be set be setup as a custom formatter in your editor,
 similar to gofmt or goimports.
 
+#### Opting-out during automatic instrumentation
+
+If you're relying on automatic instrumentation
+and want to ignore specific lines from being instrumented,
+you can add a comment in one of the following forms
+on relevant lines:
+
+```go
+//errtrace:skip
+//errtrace:skip explanation
+```
+
+This can be especially useful if the returned error
+has to match another error exactly because the caller still uses `==`.
+
+For example, if you're implementing `io.Reader`,
+you need to return `io.EOF` when you reach the end of the input.
+Wrapping it will cause functions like `io.ReadAll` to misbehave.
```

Comment:
maybe a feature request for future, should we have a set of default opt-outs inbuilt, like don't wrap `io.EOF` when being returned from `Read`? (It's not perfect, but seems like it's never correct to wrap an EOF error from `Read`)

Reply:
Yeah, we can explore that if this becomes a need in the future. Will create an issue.